app/routes/app/t/$id.tsx

```diff
@@ -0,0 +1,376 @@
+import { Search16 } from "@carbon/icons-react";
+import {
+  Input,
+  Select,
+  Title,
+  Text,
+  Button,
+  Center,
+  Divider,
+  Tabs,
+  Paper,
+  Badge,
+  Avatar,
+  Checkbox,
+  Progress,
+} from "@mantine/core";
+import { Form, Link } from "@remix-run/react";
+import type { Submission, TopicWithMarketplace } from "~/domain";
+import { Detail } from "~/components/Detail";
+import * as Author from "~/components/Author";
+import { ProjectBadge } from "~/components/ProjectBadge";
+
+export default function Challenge() {
+  return (
+    <div className="mx-auto container mb-12">
+      <section className="flex flex-col md:flex-row space-y-7 md:space-y-0 space-x-0 md:space-x-5 py-12">
+        <main className="flex-1">
+          <div className="space-y-3">
+            <Title order={2}>Challenge Title</Title>
+            <div className="flex space-x-8">
+              <Detail>
```

Comment:
getting FOUC on `Detail`. We need to remove the TW classnames on `Title` used in there